src/ol/proj/proj.js

```diff
@@ -120,6 +120,9 @@ ol.proj.Projection = function(options) {
    * @type {boolean}
    */
   this.global_ = goog.isDef(options.global) ? options.global : false;
+  if (this.global_) {
+    goog.asserts.assert(!goog.isNull(this.extent_));
```

Comment:
For "global" projections, we assume that the projection's extent is the extent of the world. So if we wrap e.g. a Swiss projection around its extent, Switzerland will be repeated several times across the globe.